app-i18n/transifex-client/transifex-client-1.1.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module
+
+DESCRIPTION="The Transifex command-line client"
+HOMEPAGE="https://github.com/transifex/cli"
+
+go-module_set_globals
+
+SRC_URI="https://github.com/transifex/cli/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	https://gitlab.com/api/v4/projects/35204985/packages/generic/${PN}/${PV}/${PN}-1.1-deps.tar.xz"
+S="${WORKDIR}"/cli-${PV}
+
+LICENSE="Apache-2.0 BSD BSD-2 ISC LGPL-3 MIT MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+src_compile() {
+	emake GOFLAGS="${GOFLAGS} -ldflags=-X="github.com/transifex/cli/internal/txlib.Version=${PV}
```

Comment:
I haven't learned how to, but we have this function called `ego` to build go stuff. Take a look for the next bump!